package.json

```diff
@@ -75,6 +79,7 @@
     "formidable": "^1.2.1",
     "ncp": "^2.0.0",
     "node-stream-zip": "^1.8.2",
+    "pirates": "^4.0.1",
```

Comment:
How will it work when we move from cjs to esmodules?

Reply:
This is a good question!

The short answer is it won't and we never will.

The long answer is
- pretty much the whole world is using pirates in some form so it probably won't be our problem to solve.
- Considering that babel + pirates gives you seamless esmodule syntax, there is no pressure to use native esmodules.
- esmodules are designed to be statically traversalable. There is code in jest to do this already, but they don't hook it up to the transpilation step because nobody needs it. But if esmodules becomes mainstream and pirates doesn't work, we could switch to that code and compile all of files upfront.
- Worst case, we switch to a vm and use `https://nodejs.org/api/vm.html#vm_class_vm_module` to hook into esmodules the same way as pirates.